cpp/es1.2/medianaEdoardo.cpp

```diff
@@ -0,0 +1,54 @@
+#include<iostream>
+#include<vector>
+
+using namespace std;
+
+vector<double> bubleSort(vector<double> n){//algoritmo di sorting
+    bool esci;
+    do{
+        esci=true;//condizione di uscita
+        for(int i=0;i<n.size()-1;i++){
+            if(n.at(i)>n.at(i+1)){//se il vector non è in ordine
+                int app=n.at(i);
```

Comment:
You could try using `swap(n[i], n[i + 1])` (see `std::swap` reference [here](https://cplusplus.com/reference/algorithm/swap/)).